add logging and system debugging section (#16)
RehanSaeed/Bash-Cheat-Sheet: README.md

free -s|--seconds 5    # Show memory usage and update continuously every five seconds
```

> ## Logs & System Debugging 

Reviewer: AI made some improvements:

```suggestion
## Logs & System Debugging

journalctl -u nginx              # Logs for a specific service
journalctl -xe                   # Recent errors with explanations
journalctl -f                    # Follow logs in real time
journalctl -k                    # Kernel logs (systemd)

dmesg | tail                     # Recent kernel messages (early boot / non-systemd)
tail -f /var/log/syslog          # Live system logs (Debian/Ubunt
```

free -s|--seconds 5    # Show memory usage and update continuously every five seconds
```

> 

Reviewer: Remove line

```suggestion
```

free -s|--seconds 5    # Show memory usage and update continuously every five seconds
```

## Logs & System Debugging

journalctl -u nginx              # Logs for a specific service
journalctl -xe                   # Recent errors with explanations
journalctl -f                    # Follow logs in real time
journalctl -k                    # Kernel logs (systemd)

dmesg | tail                     # Recent kernel messages (early boot / non-systemd)

> tail -f /var/log/syslog          # Live system logs (Debian/Ubunt
> 
> ```bash
> journalctl -u nginx              # Logs for a specific service
> journalctl -xe                   # View system errors
> journalctl -f                    # Follow logs in real time
> 
> dmesg | tail                     # Kernel logs (recent)
> tail -f /var/log/syslog          # Live system logs
> ```
> 

Reviewer: ```suggestion

## Logs & System Debugging

journalctl -u nginx              # Logs for a specific service
journalctl -xe                   # Recent errors with explanations
journalctl -f                    # Follow logs in real time
journalctl -k                    # Kernel logs (systemd)

dmesg | tail                     # Recent kernel messages (early boot / non-systemd)
tail -f /var/log/syslog          # Live system logs (Debian/Ubuntu)
tail -f /var/log/messages        # Live system logs (RHEL/CentOS)

```
